meta-rinato/recipes-kernel/linux/linux-rinato/linux-rinato_next.bb

```diff
@@ -0,0 +1,33 @@
+inherit kernel
+
+SECTION = "kernel"
+SUMMARY = "linux-next with patches for Samsung smartwatches"
+HOMEPAGE = "https://kernel.org"
+LICENSE = "GPL-2.0-only"
+LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
+COMPATIBLE_MACHINE = "rinato"
+
+SRC_URI = " \
+    git://git@github.com/casept/linux-samsung-smartwatch.git;protocol=https;branch=samsung-smartwatch \
+"
+SRC_URI[sha256sum] = "ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"
```

Comment:
Since this is a git repository (not an archive) this should not been necessary.

Reply:
Removed.